dev-libs/go-usb/go-usb-9999.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2014 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+inherit git-r3 multilib
+
+DESCRIPTION="CGO bindings for libusb"
+HOMEPAGE="https://github.com/hanwen/usb"
+EGIT_REPO_URI="https://github.com/hanwen/usb.git"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS=""
+IUSE=""
+
+DEPEND=">=dev-lang/go-1.4"
+RDEPEND=""
+
+# Tests require a connected mtp device
+RESTRICT="test"
+
+GO_PN="/usr/$(get_libdir)/go/src/github.com/hanwen/usb"
```

Comment:
Are you sure it's supposed to be libdir and not plain `lib`? go-fuse above uses `lib`. This is important for systems where lib!=lib64.